src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+
+const tasks = [
+  {
+    index: 'broom',
+    complete: true,
+    description: 'bbbbbb',
+  },
+  {
+    index: 'broom',
+    complete: true,
+    description: 'bbbbbb',
+  },
+];
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements , according to which you should have to create an array of objects. Each object does contain three properties `index`, `description` and `completed` and should be `number`, `string` and `boolean` respectively. Kindly follow the instructions correctly and do make the changes as I mentioned because your `index` is a **string** right now, not a `number`.

## Requirement
![Screenshot from 2023-08-17 21-36-08](https://github.com/Obimbo07/To-do-list/assets/66476329/fd8072c6-6e98-4a1b-b167-eb55db5e1d33)